[2025-03-17 20:38:59,335] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: weather_ingestion.transform_data scheduled__2025-03-17T19:37:56.309319+00:00 [queued]>
[2025-03-17 20:38:59,345] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: weather_ingestion.transform_data scheduled__2025-03-17T19:37:56.309319+00:00 [queued]>
[2025-03-17 20:38:59,345] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2025-03-17 20:38:59,345] {taskinstance.py:1242} INFO - Starting attempt 1 of 2
[2025-03-17 20:38:59,345] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2025-03-17 20:38:59,354] {taskinstance.py:1262} INFO - Executing <Task(PythonOperator): transform_data> on 2025-03-17 19:37:56.309319+00:00
[2025-03-17 20:38:59,357] {standard_task_runner.py:52} INFO - Started process 1904 to run task
[2025-03-17 20:38:59,363] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'weather_ingestion', 'transform_data', 'scheduled__2025-03-17T19:37:56.309319+00:00', '--job-id', '284', '--raw', '--subdir', '/Users/yellow/Desktop/data_engineering/airflow_weather_01/dags/dag_weather.py', '--cfg-path', '/var/folders/7m/b2x3pxpj4h398t57k9hlv4gm0000gn/T/tmpagqp0qaq', '--error-file', '/var/folders/7m/b2x3pxpj4h398t57k9hlv4gm0000gn/T/tmp8syl050f']
[2025-03-17 20:38:59,366] {standard_task_runner.py:77} INFO - Job 284: Subtask transform_data
[2025-03-17 20:38:59,403] {logging_mixin.py:109} INFO - Running <TaskInstance: weather_ingestion.transform_data scheduled__2025-03-17T19:37:56.309319+00:00 [running]> on host 147.0.168.192.in-addr.arpa
[2025-03-17 20:38:59,444] {taskinstance.py:1427} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=weather_ingestion
AIRFLOW_CTX_TASK_ID=transform_data
AIRFLOW_CTX_EXECUTION_DATE=2025-03-17T19:37:56.309319+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-03-17T19:37:56.309319+00:00
[2025-03-17 20:39:00,145] {python.py:152} INFO - Done. Returned value was: None
[2025-03-17 20:39:00,151] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=weather_ingestion, task_id=transform_data, execution_date=20250317T193756, start_date=20250317T193859, end_date=20250317T193900
[2025-03-17 20:39:00,183] {local_task_job.py:154} INFO - Task exited with return code 0
[2025-03-17 20:39:00,228] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
